composer.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "chapter-three/jcc-srl",
-    "description": "Judicial Council of California Self-Represented Litigant portal - Drupal 8",
+    "name": "chapter-three/jcc-tc",
+    "description": "Judicial Council of California Trial Courts - Drupal 8",
```

Comment:
Trivial detail but we should brand as despite the repo name...
"name": "judicialcouncil/jcc-base"
"description": "Judicial Council of California Web Content Management - Drupal 8"